tests/test_polydata.py

```diff
@@ -744,6 +744,10 @@ def test_tube(spline):
     tube = spline.tube(radius=0.5, scalars='arc_length', progress_bar=True)
     assert tube.n_points, tube.n_cells
 
+    # Complicated with absolute radius
+    tube = spline.tube(radius=0.5, scalars='arc_length', absolute=True, progress_bar=True)
+    assert tube.n_points, tube.n_cells
```

Comment:
These asserts along with the existing ones in the test are buggy:
```py
n_points = 1
n_cells = 0
assert n_points, n_cells
# no error
```
I think we can get away with
```suggestion
    assert tube.n_points and tube.n_cells
```
but please fix it in the rest of this test :)

Reply:
Great catch, thank you!